maker/src/cli.rs

```diff
@@ -17,6 +17,28 @@ pub struct Opts {
     /// Where to permanently store data, defaults to the current working directory.
     #[clap(long)]
     data_dir: Option<PathBuf>,
+
+    #[clap(value_enum, default_value = "regtest")]
+    pub network: Network,
+}
+
+#[derive(Debug, Clone, Copy, clap::ValueEnum)]
+pub enum Network {
+    Regtest,
+    Signet,
+    Testnet,
+    Mainnet,
+}
+
+impl From<Network> for bitcoin::Network {
+    fn from(network: Network) -> Self {
+        match network {
+            Network::Regtest => bitcoin::Network::Regtest,
+            Network::Signet => bitcoin::Network::Signet,
```

Comment:
❓ Should we add more information to the `Signet` and `Regtest` variants to identify the actual instance of `signet` or `regtest`?

Reply:
I guess you are referencing to e.g. what coordinator endpoint, pubkey, etc. to use? We can add this in a follow up PR.